script/Ciel/Day 12/12_01_CIEL12_3_1.txt

```diff
@@ -156,6 +156,7 @@ Why do they act this way?
 [sha:3f7012fc4e60100f99c55bc9c915c018e97e253c]{
 -- Page 163, Offset 31292.
 -- 無為な昆虫のように、知性あるものが、命題もなく群れているのか。
+// Petrikow flag for contrast
 Why do these intelligent beings mill about, without purpose, like shiftless insects?
```

Comment:
Something like this was my thought...
Essentially, "why act like insect (dumb) when smart?"
>Why are they milling about, without purpose, like shiftless insects, when endowed with the gift of wisdom?